### PR TITLE
Adds Google Cast Support

### PR DESCRIPTION
**Pull request by redreceipt**
Lays the groundwork for Google Cast support. 

Component 1

*NOTE:* This requires the template to enable Android Multidex. That is included (and working) in this PR.

### CODE REVIEW

File: apolloschurchapp/ios/apolloschurchapp/AppDelegate.m [hunk 1]
```diff
@@ -42,14 +43,19 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
+  GCKCastOptions* options = [[GCKCastOptions alloc] initWithDiscoveryCriteria:criteria];
+  [GCKCastContext setSharedInstanceWithOptions:options];
+
   rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
 
   self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
-  
+
```
Comment (redreceipt) [inline apolloschurchapp/ios/apolloschurchapp/AppDelegate.m]:
```suggestion
  
```

File: apolloschurchapp/ios/apolloschurchapp/AppDelegate.m [hunk 2]
```diff
@@ -42,14 +43,19 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
   rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
 
   self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
   
   [RNBootSplash initWithStoryboard:@"BootSplash" rootView:rootView];
-  
+
```
Comment (redreceipt) [inline apolloschurchapp/ios/apolloschurchapp/AppDelegate.m]:
```suggestion
  
```

**Comment by redreceipt** [comment]
this will stay a draft until I can get a screenshot

**Comment by redreceipt** [comment]
Ok this one is ready

**Comment by conrad-vanl** [comment]
This is good to be merged as soon as `apollos-apps` has a release including google cast

**Comment by burkeshartsis** [comment]
@conrad-vanl @redreceipt can/does this need to be merged?

**Comment by redreceipt** [comment]
Yep I believe so. The apps PR was merged earlier today.

**Comment by vinnyjth** [comment]
@conrad-vanl In the past, we have defined the linked state as the default state for this repo, and required forking from a release tag if you want things to work out of the box. I think this is invalid, and we should probably flip the paradigm and work from a `development` branch and then merge development master for releases. For now though, let's merge this into master until we have an alternative better outlined.

**Comment by vinnyjth** [comment]
@conrad-vanl Actually, I see no reason not to give the `develop` branch a shot :)

**Pull request merged**